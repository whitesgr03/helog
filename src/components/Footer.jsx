import style from "../styles/Footer.module.css";
import { container } from "../styles/Contact.module.css";
import image from "../styles/utils/image.module.css";

import Address from "../components/Address";

const Footer = () => {
	return (
		<footer className={style.footer}>
			<div className={container}>
				<a href="#git" alt="github" className={style.link}>
					<span className={`${image.icon} ${style.github}`} />
				</a>
				<div className={style.address}>
					<Address />
				</div>
			</div>
			<p>&copy; 2024 Designed &amp; coded by Weiss Bai</p>
		</footer>
	);
};

export default Footer;
