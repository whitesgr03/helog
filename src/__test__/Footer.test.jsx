import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";

import Footer from "../components/Footer";

describe("Footer component", () => {
	it("should render correctly", () => {
		const { asFragment } = render(<Footer />);

		const actual = asFragment();

		expect(actual).toMatchSnapshot();
	});
});
