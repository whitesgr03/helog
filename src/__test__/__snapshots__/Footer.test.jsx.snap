// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > should render correctly 1`] = `
<DocumentFragment>
  <footer
    class="_footer_4d8704"
  >
    <div>
      <a
        alt="github"
        class="_link_4d8704"
        href="#git"
      >
        <span
          class="_icon_7e06e0 _github_4d8704"
        />
      </a>
      <div
        class="_address_4d8704"
      >
        <div>
          Address component
        </div>
      </div>
    </div>
    <p>
      Â© 2024 Designed & coded by Weiss Bai
    </p>
  </footer>
</DocumentFragment>
`;
